.github/workflows/ci.yml

```diff
@@ -73,23 +72,24 @@ jobs:
       # We could run 'make lint' but we prefer this action because it leaves
       # 'annotations' (i.e. it comments on PRs to point out linter violations).
       - name: Lint
-        uses: golangci/golangci-lint-action@v3
+        uses: golangci/golangci-lint-action@1481404843c368bc19ca9406f87d6e0fc97bdcfd # v7.0.0
         with:
-          version: ${{ env.GOLANGCI_VERSION }}
+          version: v2.1.1
+          args: --timeout 5m
```

Comment:
We could revert back to the `env` here, as I removed the renovate marker. And maybe also not pin the version. Depending on how much we want to differ the upstream.

Reply:
Unpinning also works for me. Even simpler. I am pretty sure the action runs the "latest" version by default.